shared-components/package.json

```diff
@@ -4,12 +4,15 @@
   "private": true,
   "type": "module",
   "description": "Shared React components of Boxtribute FE Apps",
-  "main": "index.js",
   "scripts": {
+    "build": "pnpm generate-graphql-ts-types && tsc",
```

Comment:
I imagine this was the solution for that CI error about `tsc` and GraphQL types. GJ! 🎉

Reply:
It was actually adding the reference to the dependency directly (same commit) that did it, I just added this so I could ensure it builds at the component level.